nodejs/awsx/apigateway/README.md

```diff
@@ -94,6 +94,57 @@ let endpoint = new awsx.apigateway.API("example", {
 
 A complete example can be found [here](https://github.com/pulumi/pulumi-awsx/blob/master/nodejs/awsx/examples/api/index.ts).
 
+#### Integration Route
+
+An Integration Route is a route that will map an endpoint to a specified backend. The supported types are:
+
+    * `http` or `http_proxy`: integration with an HTTP backend
+    * `aws_proxy`: integration with AWS Lambda functions
+    * `aws`: integration with AWS Lambda functions or other AWS services, such as Amazon DynamoDB, Amazon Simple Notification Service or Amazon Simple Queue Service
+    * `mock`: integration with API Gateway without invoking any backend
+
+```ts
+import * as awsx from "@pulumi/awsx";
+
+let endpoint = new awsx.apigateway.API("example", {
+    routes: [{
+        path: "/integration",
+        target: {
+            uri: "https://www.google.com",
+            type: "http_proxy",
```

Comment:
if it's http_proxy, do you need to create a vpc_link: https://docs.aws.amazon.com/apigateway/latest/developerguide/getting-started-with-private-integration.html  ?  or is that optional for an `http_proxy`?

Reply:
i think its optional since it works as is